flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BDREHDFSEventSink.java

```diff
@@ -67,7 +67,7 @@
   private static final String defaultInUsePrefix = "";
   private static final String defaultInUseSuffix = ".tmp";
   private static final long defaultBatchSize = 100;
-  private static final String defaultFileType = HDFSWriterFactory.SequenceFileType;
+  private static final String defaultFileType = HDFSWriterFactory.SEQUENCE_FILE_TYPE;
   private static final int defaultMaxOpenFiles = 5000;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00115)